utils/localToEmoji.ts

```diff
@@ -0,0 +1,17 @@
+export default function localToEmoji(local: string): string {
+  switch (local) {
+    case "en":
+    case "en_gb":
+    case "en_ca":
```

Comment:
Can we add Australia and New Zealand (`en_au` and `en_nz`), or just match all `en_*`

Reply:
that was mostly just testing on my end, in the future they will just have their own emoji instead of using the US one